ciao-controller/controller_test.go

```diff
@@ -742,7 +723,12 @@ func testStartTracedWorkload(t *testing.T) *testutil.SsntpTestClient {
 		t.Fatal(err)
 	}
 
-	client := newTestClient(0, ssntp.AGENT)
+	client, err := testutil.NewSsntpTestClientConnection("StartTracedWorkload", ssntp.AGENT)
+	if err != nil {
+		t.Fatal(err)
+	}
+	// deliberately leak
+	//defer client.Ssntp.Close()
 
```

Comment:
ah ok, I reviewed the code, I did want to have the caller close the connection when they were done with it.